include/acl_selectors.php

```diff
@@ -182,10 +182,10 @@ function contact_select($selname, $selclass, $preselected = false, $size = 4, $p
 	}
 
 	if($privmail) {
-		$sql_extra .= " AND `network` IN ( 'dfrn', 'dspr' ) ";
+		$sql_extra .= " AND `network` IN ( NETWORK_DFRN, NETWORK_DIASPORA ) ";
```

Comment:
This will not work: the constant isn't replaced when inside a string.
you need something like

```php
$sql_extra ,= sprintf(" AND `network` IN ( '%s' , '%s' ) ", NETWORK_DFRN, NETWORK_DIASPORA);
```

same on line 188